### PR TITLE
enhance readme structure

### PR DESCRIPTION
**Pull request by schmierkov**
This PR supposed to enhance the first impression of the Readme. Please tell me what you think about it!
Cheers

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/DatabaseCleaner/database_cleaner.svg?branch=master)](https://travis-ci.org/DatabaseCleaner/database_cleaner) [![Code Climate](https://codeclimate.com/github/DatabaseCleaner/database_cleaner/badges/gpa.svg)](https://codeclimate.com/github/DatabaseCleaner/database_cleaner)
+
```
Comment (etagwerker) [inline README.markdown]: @schmierkov The badges before the title look weird. Please put it after the title.

**Comment by etagwerker** [comment]
@schmierkov Other than that it looks good.

**Comment by schmierkov** [comment]
@etagwerker done, check it out!

**Comment by etagwerker** [comment]
@schmierkov Great, thanks! 🍻

**Pull request merged**